include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -527,11 +533,20 @@ namespace internal
       }
   }
 
-  // This a specialization for Gauss-Lobatto elements where the 'values'
-  // operation is identity, which allows us to write shorter code.
-  template <int dim, int fe_degree, int n_q_points_1d, int n_components, typename Number>
-  struct FEEvaluationImpl<MatrixFreeFunctions::tensor_gausslobatto, dim,
-    fe_degree, n_q_points_1d, n_components, Number>
+
+
+  /**
+   * This struct performs the evaluation of function values, gradients and
+   * Hessians for tensor-product finite elements.  This a specialization for
+   * symmetric basis functions with the same number of quadrature points as
+   * degrees of freedom. In that case, we can first transform to a spectral
+   * basis and then perform the evaluation of the first and second derivatives
```

Comment:
This is what some people refer to as "spectral element" in the proper sense. Other call it collocation (same location of node points and quadrature). I can also replace `Spectral` in the name by `Collocation` if you prefer.